graql/executor/property/ValueTypeExecutor.java

```diff
@@ -51,16 +51,16 @@
         }
         this.property = property;
 
-        if (!VALUE_TYPES.containsKey(property.valueType())) {
+        if (!VALUE_TYPES.containsKey(property.ValueType())) {
```

Comment:
The method name looks weird here, shouldn't it be camel case and thus small `v` at the start?

Reply:
yeah i made a mistake, ive already fixed it in my next "actual" PR in `graql` that will go in with the implicit relations

Reply:
didn't think it was worth a wholew new PR